src/ansiblelint/rules/jinja.py

```diff
@@ -844,17 +847,17 @@ def test_jinja_transform(
         matches = runner_result.matches
         assert len(matches) == 2
 
-        orig_dir, tmp_dir = copy_examples_dir
-        orig_playbook = orig_dir / playbook
-        expected_playbook = orig_dir / playbook.replace(".yml", ".transformed.yml")
-        transformed_playbook = tmp_dir / playbook
-
-        orig_playbook_content = orig_playbook.read_text()
-        expected_playbook_content = expected_playbook.read_text()
-        transformed_playbook_content = transformed_playbook.read_text()
+        orig_content = playbook.read_text(encoding="utf-8")
+        expected_content = playbook.with_suffix(
+            f".transformed{playbook.suffix}",
+        ).read_text(encoding="utf-8")
+        transformed_content = playbook.with_suffix(f".tmp{playbook.suffix}").read_text(
+            encoding="utf-8",
+        )
 
-        assert orig_playbook_content != transformed_playbook_content
-        assert transformed_playbook_content == expected_playbook_content
+        assert orig_content != transformed_content
+        assert expected_content == transformed_content
+        playbook.with_suffix(f".tmp{playbook.suffix}").unlink()
```

Comment:
Hmm, these should probably be in a `finally` block to make sure they get cleaned up if the test aborts?

Reply:
I agree with you about this but we will leave this for a follow-up. For the moment I added these files to git ignore.